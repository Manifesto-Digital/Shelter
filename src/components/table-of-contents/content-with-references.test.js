import { snapshotComponent } from 'test-helpers';
import React from 'react';
import ContentWithReferences from './content-with-references';
import {
  createChildContentfulRichText,
  createFactory,
} from '../../utils/test-factories';

export const createTableOfContents = createFactory({
  tableOfContents: [
    {
      textContent: createChildContentfulRichText(),
      title: 'Public Order Act 1986',
    },
    {
      textContent: createChildContentfulRichText(),
      title: 'Police and Criminal Evidence Act 1984',
    },
  ],
});

it('renders correctly', () => {
  const mockData = createTableOfContents();
  snapshotComponent(
    <ContentWithReferences {...mockData} updateReferenceList={jest.fn()} />
  );
});
