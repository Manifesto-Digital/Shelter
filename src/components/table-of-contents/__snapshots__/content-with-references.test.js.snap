// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
.c0 {
  margin: 60px auto;
}

.c2 {
  margin-bottom: 30px;
}

.c1 {
  margin-bottom: 40px;
}

<div
  className="c0"
>
  <div
    className="c1"
  >
    <h3
      className="c2"
      id="title-0"
    >
      Public Order Act 1986
    </h3>
  </div>
  <div
    className="c1"
  >
    <h3
      className="c2"
      id="title-1"
    >
      Police and Criminal Evidence Act 1984
    </h3>
  </div>
</div>
`;
